src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -78,25 +81,17 @@ private void extractAuthor() {
      * property filled with jan will override this behavior and no data is lost. In the cases, where xmp is written by
      * another service, the assumption is, that the 1st January is not a publication date at all.
      */
```

Comment:
Just saw this, I guess the javadoc needs to be updated to reflect the new behavior

Reply:
Ah, you're right, and it looks I've forgotten to change the description. Will change that as part of the last task remaining

Reply:
@Siedlerchr any idea which document I have to edit?
I know https://github.com/JabRef/user-documentation/tree/main/en contains the docs but I'm not sure which doc to edit. Some docs related to what I'm doing are:-
https://github.com/JabRef/user-documentation/blob/main/en/advanced/xmp.md
https://github.com/JabRef/user-documentation/blob/main/en/advanced/fields.md
and maybe this : https://github.com/JabRef/user-documentation/blob/main/en/collect/findunlinkedfiles.md

But I'm not sure which one I should edit

Reply:
The xmp one should be the only relevant one. The findunlinked files is a different feature,

Reply:
>  https://github.com/JabRef/user-documentation/blob/main/en/advanced/xmp.md

Okay I looked into the doc and there's no warning note, or any in particular, regarding JabRef's previous inability to convert day from XMP to bibtex